src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.Tests/inputs/DependencyGraph/input/core-sdk/c57bde45936fc120c0124bf964f570c45bfd8f8b/graph-with-toolset.xml

```diff
@@ -242,6 +242,13 @@
       <Type>Toolset</Type>
       <RepoUri>Extensions</RepoUri>
       <Commit>0de62ae930da31048ba7e54c8cd0c6c0bcbd9095</Commit>
+    </Dependency>
+	<Dependency>
+      <Name>Microsoft.AspNetCore.BenchmarkRunner.Sources</Name>
```

Comment:
Where is this new dependency used?
Indentation also looks weird

Reply:
It was somewhere under aspnet. The issue was that we were determining all unique dependencies without regards to the dependency type.  When I switched to doing a breadth-first traversal vs. depth first this dependency, which appears as both types, switched from product to toolset.  So, I changed the algorithm to take type into account, then added this dependency to the list of what we expect.